Gems/BarrierInput/Code/Include/BarrierInput/RawInputNotificationBus_Barrier.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
```

Comment:
All of the headers were fixed recently to split this into 2 lines to avoid problems with clang formatting. Should be:

/*
 * Copyright (c) Contributors to the Open 3D Engine Project. 
 * For complete copyright and license terms please see the LICENSE at the root of this distribution.
 *
 * SPDX-License-Identifier: Apache-2.0 OR MIT
 */

Reply:
Sorry, I created these files when we still had the old copyright header, updated all files in the PR to use the new one.